UPGRADING.rst

```diff
@@ -11,6 +11,54 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#4190 Create SNS topic for monitoring and security notifications
+================================================================
+
+A new environment variable called ``AZUL_NOTIFICATION_EMAIL`` has been added. In
+personal deployments, set this variable to ``'{AZUL_OWNER}'``. As
+always, use the sandbox deployment's ``environment.py`` as a model when
+upgrading personal deployments.
+
+Note: The SNS topic and email subscription will only be created for deployments
+that have ``AZUL_ENABLE_MONITORING`` enabled, which is typically the case in
+main deployments only.
+
+**IMPORTANT**: The SNS topic subscription will be created with a status of
+"pending confirmation". Instead of simply clicking the link in the "Subscription
+Confirmation" email, you should follow the instructions given during the
+``make deploy`` process to confirm the subscription.
+
+Operator
+~~~~~~~~
+
+The first time this change is deployed to ``dev``, ``anvildev``, ``prod``, and
+``anvilprod`` the ``deploy`` job will fail in GitLab due to the confirmation
+step requiring user input at the terminal. Follow these steps to complete the
+deploy::
+
+1. Contact the recipients of ``AZUL_NOTIFICATION_EMAIL`` with instructions to
```

Comment:
This should precede the deploy and be part of the PR checklist. In general work as much of this as possible into the checklist so that this and the checklist don't give contradictory information.